Components/SettingsBar/SettingsBarItem.tsx

```diff
@@ -0,0 +1,15 @@
+import { MouseEventHandler } from "react";
+
+const SettingsBarItem = ( { itemName, iconName, isActive, onClick } : { itemName:string, iconName:string, isActive:boolean, onClick:MouseEventHandler} ) => {
+  return (
+    <div 
+      className={"flex items-center hover:bg-zinc-100 py-2 px-2 cursor-pointer rounded-md text-sm".concat(isActive?" text-zinc-900 bg-zinc-100":" text-zinc-700 bg-white")}
+      onClick={onClick}
+    >
+      <span className="material-icons-outlined">{iconName}</span>
+      <div className="pl-2">{itemName}</div>
```

Comment:
Use `font-inter` and also check if we can have icons with thin widths.

Reply:
Sir, I think with `Material Symbols` we can control the stroke widths through font-weight.
I made commit for `font-inter` sir, very minor change only

Reply:
@PiyushKalyanpy Sir, 
If it is okay for me adding `Material-Symbols` a similar package to `Material-Icons` to the project. Then we can control the icon stroke width. Let me know your thoughts on this.